### DISCUSSION TITLE
STM32F7DISCO with LCD _ How update the firmware with lcd (jimmo database)?

### DISCUSSION THREAD

**Discussion by IHOXOHI** [micropython/micropython#13426] [STM32 / Pyboard]
Hi,
I would like to use uasyncio on the stm32F7DISCO with the library https://github.com/jimmo/micropython.git
...?
I'm looking for find which command could I use for control the lcd, too...  
Thanks a lots.

**Comment by Josverl** [comment]
I think you may have things mixed up, the repo you link to is a fork of the [MicroPython Repo](https://github.com/micropython/micropython)

**Reply by IHOXOHI** [reply]
Yes.
In this case is there a way for upgrade, at least add uasyncio?
Thanks.
